configuration/views.py

```diff
@@ -53,6 +69,58 @@ def get_serializer_class(self):
         else:
             return super().get_serializer_class()
 
+    @action(
+        detail=False,
+        methods=["GET"],
+        serializer_class=UnConStudiesConfigurationSerializer,
+        permission_classes=[AllowAny],
+    )
+    def uncon_studies_form(self, request, **kwargs):
+        existing_journals = (
+            Study.objects.values("abbreviated_source_title")
+            .order_by("abbreviated_source_title")
+            .distinct()
+            .values_list("abbreviated_source_title", flat=True)
+        )
+        available_populations_types = UnConSampleChoices.values
+
+        available_consciousness_measure_phase_type = UnConsciousnessMeasurePhase.objects.all()
+        available_consciousness_measure_type = UnConsciousnessMeasureType.objects.all()
+        available_consciousness_measure_sub_type = UnConsciousnessMeasureSubType.objects.all()
+        available_tasks_types = UnConTaskType.objects.all()
+        available_processing_sub_domain_types = UnConProcessingSubDomain.objects.all()
+        available_processing_main_domain_types = UnConProcessingMainDomain.objects.all()
+        available_main_paradigm_type = UnConMainParadigm.objects.all()
+        available_authors = Author.objects.all()
+        available_stimulus_modality_type = UnConModalityType.objects.all()
+        available_stimulus_category_type = UnConStimulusCategory.objects.all()
+        available_stimulus_sub_category_type = UnConStimulusSubCategory.objects.all()
+        available_experiment_types = [dict(name=v, value=k) for k, v in ExperimentTypeChoices.choices]
```

Comment:
it's confusing to use `name, value` instead of `key, value`, and more confusing when `value=k`

Reply:
it's what the front is expecting so I needed to adapt my self to name, value